README.md

```diff
@@ -82,7 +82,17 @@ val settings: HttpMetricsSettings = HttpMetricsSettings
 val registry: HttpMetricsRegistry = ... // concrete registry implementation
 
 val route: Route = ... // your route
+```
+
+You can bind this route to a HTTP server as follows
+
+```scala
+//If you are using akka-http >= 10.2.0
+Http().newServerAt("localhost", 8080).bindFlow(route.recordMetrics(registry))
+```
 
+```scala
+//If you are using akka-http < 10.2.0
```

Comment:
This should go away. The new release will only be binary compatible with akka-http 10.2

Reply:
Gotcha, I added that just in case some had to lock into a previous version of akka-http-metrics in case they are unable to upgrade yet to akka-http 10.2+, they can of course always look at tagged README in that event though.